test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
Great job @Jaymelfah 🤝

- [ ] As specified in the project requirements, kindly add the `test.md` file to the `.gitignore` file. This will allow it not to be present on Github.

Reply:
@1GORDON. 
I realised that since i created all the files straight in my github workspace and not locally i am unable to get the gitignore file to work. I have created a new branch locally called "new-feature" where i have been able to include all the changes you suggested. However I have not created a new pull request since it has not been demanded to do so.

Reply:
You don't need to create a new PR. Just update your PR. The changes will reflect on the branch after you have committed the changes. Forgive me for the late response.✌